components/Typography/src/MDCTypography.h

```diff
@@ -42,6 +42,21 @@
 /** Asks the receiver to return an italic font. */
 - (nonnull UIFont *)italicFontOfSize:(CGFloat)fontSize;
 
+@optional
```

Comment:
actually the light, medium, and regular are required. italic and bold are not. I will move the optional to there.